bumble/device.py

```diff
@@ -5057,6 +5072,15 @@ def on_flush(self):
     # [Classic only]
     @host_event_handler
     def on_link_key(self, bd_addr, link_key, key_type):
+        authenticated = key_type in (
+            hci.HCI_AUTHENTICATED_COMBINATION_KEY_GENERATED_FROM_P_192_TYPE,
+            hci.HCI_AUTHENTICATED_COMBINATION_KEY_GENERATED_FROM_P_256_TYPE,
+        )
+        pairing_keys = PairingKeys()
+        pairing_keys.link_key = PairingKeys.Key(
+            value=link_key, authenticated=authenticated
+        )
```

Comment:
Seems like a repeat of `if self.keystore:...`

Reply:
Fixed